pkg/cluster/kubeadm/config.go

```diff
@@ -110,18 +156,24 @@ func Config(data ConfigData) (config string, err error) {
 	if err != nil {
 		return "", err
 	}
-	// The complexity of the config does not require special handling
-	// between v1alpha1 and v1alpha2 yet.
-	templateSource := DefaultConfigTemplateAlphaV3
-	if ver.LessThan(version.MustParseSemantic("v1.12.0")) {
-		templateSource = DefaultConfigTemplateAlphaV1orV2
+
+	templateSource := ConfigTemplateBetaV1
+	if ver.LessThan(version.MustParseSemantic("v1.13.0")) {
```

Comment:
This is wrong (this logic needs to be in the opposite order, lowest check first), will fix when I get back to a computer.

Reply:
updated